index.html

```diff
@@ -178,10 +178,10 @@ <h3>Common Voice</h3>
 			<h2>Spaces</h2>
 			<p>Find a local space or meetup where you can work with like-minded people!</p>
 			<div class="row">
-				<a class="col-md-4" href="https://www.meetup.com/opensourceladies/" target="_blank">
+				<a class="col-md-4" href="https://www.meetup.com/opensourcediversity/" target="_blank">
 					<img alt="" src="img/projects/opensourceladies.jpg" />
-					<h3>Open Source Ladies</h3>
-					<p>Monthly meetup in Berlin for women* who want to contribute to open source projects.</p>
+					<h3>Open Source Diversity</h3>
```

Comment:
Based on the conversation we had at #1 Open Source Ladies will be renamed to Open Source Diversity Berlin.